src/test/cli.ts

```diff
@@ -98,6 +99,8 @@ async function runTests(args: string[], opts: { [key: string]: any }) {
   const overrides = overridesFromOptions(opts);
   if (opts.headed)
     overrides.use = { headless: false };
+  if (opts.baseURL)
+    overrides.use = { baseURL: opts.baseURL };
```

Comment:
Looks like headed above will get real upset if this code runs.

Reply:
Yeah, you're right. I've updated it to fix it.